content/blog/customer-managed-agents-kubernetes/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: "Kubernetes-native Option for Pulumi Deployments Customer-Managed Agents"
+allow_long_title: true
+date: 2024-07-29T09:08:31-07:00
+draft: false
+meta_desc: Introducing Kubernetes-native support for Customer Managed Agents in Pulumi Cloud for enhanced flexibility and control over your infrastructure deployments.
+meta_image: meta.png
+authors:
+    - levi-blackstone
+    - meagan-cojocar
+tags:
+    - releases
+---
+
+We are excited to announce the availability of Kubernetes-native support for Customer-Managed Agents for Pulumi Deployments, further enhancing the flexibility and control over your infrastructure deployments. This new feature allows you to self-host deployment agents within your Kubernetes environment, bringing the same power and flexibility of Pulumi-hosted deployments to isolated environments.
+
+## Customer Managed Agents: Flexibility and Control
```

Comment:
Meta comment, technically we supported k8s before, but it was not widely circulated and most k8s admins would reject as it require pod security elevation, do we want to talk to that?

Reply:
Not really as I think it makes it more exciting to focus on what is new, but let me know if you think its important context to include.